### PR TITLE
update pull request template to be similar to godotengine/godot

### PR DESCRIPTION
**Pull request by nathanfranke**
\

### CODE REVIEW

**Review by aaronfranke — APPROVED** [review]
See this PR in the main repo: https://github.com/godotengine/godot/pull/62486

**Review by YuriSizov — CHANGES_REQUESTED** [review]
We don't have 3.x here, you must use a specific version branch, the one that corresponds to the latest applicable version. So the text must be adjusted.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <!--
-**Note:** Pull Requests should be made against the `master` by default.
-
-Only make Pull Requests against other branches (e.g. `2.1`) if your changes only apply to that specific version of Godot.
-
-The type of content accepted into the documentation is explained here: https://docs.godotengine.org/en/latest/community/contributing/content_guidelines.html
+Please target the `master` branch in priority.
+PRs can target other branches if the same change was done in `master`, or is not relevant there.
```
Comment (YuriSizov) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
PRs can target other branches (e.g. `3.2`, `3.5`) if the same change was done in `master`, or is not relevant there.
PRs must not target `stable`, as that branch is updated manually.
```

I'd still keep an example, and also mention that stable is not to be used.

**Review by YuriSizov — CHANGES_REQUESTED** [review]
Slight adjustment and another note worth pointing out, and then I think this is fine.

**Review by YuriSizov — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**